.goreleaser.yml

```diff
@@ -83,26 +85,16 @@ dockers:
       - config/default.yml
 
 docker_manifests:
-  - name_template: "{{ if .IsNightly }}flipt/flipt:nightly{{ else }}flipt/flipt:latest{{ end }}"
-    image_templates:
-      - "{{ if .IsNightly }}flipt/flipt:{{ incpatch .Version }}-nightly-amd64{{ else }}flipt/flipt:{{ .Tag }}-amd64{{ end }}"
-      - "{{ if .IsNightly }}flipt/flipt:{{ incpatch .Version }}-nightly-arm64{{ else }}flipt/flipt:{{ .Tag }}-arm64{{ end }}"
   - name_template: "{{ if .IsNightly }}flipt/flipt:{{ incpatch .Version }}-nightly{{ else }}flipt/flipt:{{ .Tag }}{{ end }}"
     image_templates:
       - "{{ if .IsNightly }}flipt/flipt:{{ incpatch .Version }}-nightly-amd64{{ else }}flipt/flipt:{{ .Tag }}-amd64{{ end }}"
       - "{{ if .IsNightly }}flipt/flipt:{{ incpatch .Version }}-nightly-arm64{{ else }}flipt/flipt:{{ .Tag }}-arm64{{ end }}"
-  - name_template: "{{ if .IsNightly }}markphelps/flipt:nightly{{ else }}markphelps/flipt:latest{{ end }}" # TODO: deprecate
```

Comment:
gah, this does break the `nightly` tag tho..

Reply:
I think I fixed it by breaking out nightly into its own `.goreleaser.nightly.yml` and using the [includes](https://goreleaser.com/customization/includes/?h=include) functionality of goreleaser